types/node/node-tests.ts

```diff
@@ -976,10 +976,11 @@ async function asyncStreamPipelineFinished() {
         // close callback parameter is optional
         _server = _server.close();
 
-        // close callback parameter doesn't specify any arguments, so any
-        // function is acceptable
+        // close callback parameter can be either nothing (undefined) or an error
         _server = _server.close(() => { });
-        _server = _server.close((...args: any[]) => { });
+        _server = _server.close((err) => {
+            if (err != null) { const _err: Error = err; }
```

Comment:
According to type definiton `err` is either an `Error` or `undefined` therefore a check against `null` seems to be wrong here

Reply:
Good point @Flarna. `!= null` is generally used to check not `null` or `undefined` (but not other falsy values). I agree that it's not precise. Thanks for pointing it out.

Addressed in 4809757.